src/index.js

```diff
@@ -112,7 +112,7 @@ function cartItemsReducer(cartItems, action) {
   }
 }
 
-export const CartContext = createContext([{}, () => {}]);
+export const CartContext = createContext([{ cartItems: [] }, () => {}]);
```

Comment:
I'd assume that if we're making sure there's a `cartItems` when there's no `CartProvider` that we should make sure everything else has its default value. You may want to define a variable above that has the default values for the rest of the cart (stuff handled by `cartReducer`) to avoid duplicating a bunch of lines in the code.

```suggestion
export const CartContext = createContext([{
  lastClicked: '',
  shouldDisplayCart: false,
  stripe,
  successUrl,
  cancelUrl,
  currency,
  billingAddressCollection,
  allowedCountries,
  skus: {},
  cartItems: [],
}, () => {}]);
```

Reply:
Oh! How obvious, can't believe I didn't think of that lol. Gonna test this out now